code/resources/js/components/Footer.vue

```diff
@@ -0,0 +1,22 @@
+<template>
+    <div>
+    <div class="footer">
+            <div class="content has-text-centered">
+                <p>
+                    <strong>MySuperTasker</strong> by <a href="https://github.com/Niko998" color="black">Nikodem Kabsch</a>.
+                </p>
+            </div>
+        </div>
+        </div>
+</template>
+
+<style>
+.footer{
+    position: absolute;
```

Comment:
dlaczego position absolute? W bulmie jeśli dobrze poustawiasz style na całej stronie to footer wyląduje tam gdzie powinien.